Comp2024/src/main/java/frc/robot/subsystems/CommandSwerveDrivetrain.java

```diff
@@ -143,16 +143,8 @@ public void periodic( )
     }
   }
 
-  // public PathPlannerPath driveToSpeaker(CommandSwerveDrivetrain drivetrain)
-  // {
-  //   in progress
-
-  //   Pose2d speakerEndGoal = (1.4,5.52,0)
-  //   PathConstraints speakerConstraints = 
-
-  //   PathPlannerPath driveToSpeaker = new PathPlannerPath (// odometry current pose, constraints, end goal);
-  //   return driveToSpeaker;
-
-  // }
-
+  public Command drivePathtoPose(CommandSwerveDrivetrain drivetrain, Pose2d pose)
+  {
+    return AutoBuilder.pathfindToPoseFlipped(pose, PATHconsts.constraints, 0, 0.0);
```

Comment:
Is there a reason we are using the API that specifies the rotational delay? (I think having the 0.0 end goal velocity makes sense).